cmd/sysl/cmd_mod.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/anz-bank/sysl/pkg/mod"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type modCmd struct {
+	subcommand string
+}
+
+func (m *modCmd) Name() string       { return "mod" }
+func (m *modCmd) MaxSyslModule() int { return 0 }
```

Comment:
What does this function do?

Reply:
No idea. All the cmd functions include it. I think it's got something to do with how many sysl modules can be passed into the commandline.